system-contract-dapp-playground/src/sections/landing/index.tsx

```diff
@@ -21,10 +21,79 @@
  */
 
 import { motion } from 'framer-motion';
+import { useToast } from '@chakra-ui/react';
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/navigation';
+import { storeAccountsInCookies } from '@/api/cookies';
 import { VerticalCommonVariants } from '@/libs/framer-motion/variants';
+import { requestAccount, isCorrectHederaNetwork, getWalletProvider } from '@/api/wallet';
+import { NoEthToast, CommonErrorToast, NetworkMismatchToast } from '@/components/toast/CommonToast';
 
 const LandingPage = () => {
+  const router = useRouter();
+  const toaster = useToast();
+  const [accounts, setAccounts] = useState<string[]>([]);
   const verticalVariant = VerticalCommonVariants(30, 0.5);
+  const { walletProvider, err: walletProviderErr } = getWalletProvider();
+
+  /** @dev handle connect wallet when a user click `connect wallet` button */
+  const handleConnectWallet = async () => {
+    // handle ethereum object or walletProvider being null by toasting it out on the client
+    if (walletProviderErr === '!ETHEREUM' || !walletProvider) {
```

Comment:
Is this string hardcoded to Metamask?
Should it be Hedera?

Either ways pull it out to a const of a class variable

Reply:
Oh this is customized by me. You can find it here https://github.com/hashgraph/hedera-smart-contracts/blob/dapp-connect-wallet-feat/system-contract-dapp-playground/src/api/wallet/index.ts#L46

I'll switch it to Hedera!